### PR TITLE
Fixes #9121: better readme for the github project page

### PR DESCRIPTION
**Pull request by dmitri-d**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+# Supported Modules
+Currently Supported modules:
+ * BMC - BMC management of devices supported by freeipmi and ipmitool
+ * DHCP - ISC DHCP and MS DHCP Servers
+ * DNS - Bind and MS DNS Servers
+ * Puppet - Any Puppet server from 0.24.x
+ * Puppet CA - Manage certificate signing, cleaning and autosign on a Puppet CA server
+ * Realm - Manage host registration to a realm (e.g. FreeIPA)
+ * TFTP - any UNIX based tftp server
```
Comment (ohadlevy) [inline README.md:25]: libvirt?

reference to other plugins? / talk about plugin arch etc?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+# Installation
+Read the [Smart Proxy Installation section](http://theforeman.org/manuals/1.8/index.html#4.3.1SmartProxyInstallation) of the manual.
+
+# Configuration
+Read the [Smart Proxy Settings section](http://theforeman.org/manuals/1.8/index.html#4.3.2SmartProxySettings) of the manual.
+
+# For Developers
+* [API Reference](http://projects.theforeman.org/projects/smart-proxy/wiki/API)
+* Smart Proxy Plugin development [how-to] (http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Smart-Proxy_Plugin)
```
Comment (ohadlevy) [inline README.md:35]: oh, missed it.

File: README.md (hunk 1 above)
Comment (dmitri-d) [inline README.md:25]: Is it worth talking about libvirt here? Install docs (referenced below) cover that install scenario...

**Comment by logicminds** [comment]
I actually started a similar development doc but haven’t contributed this yet.  Here is what I wrote when I was working on a some BMC stuff.  I can submit a PR later today.  Or feel free to just add what I wrote to this PR.

https://gist.github.com/logicminds/d387d27d273374797c3b

File: README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+[![Build Status](http://ci.theforeman.org/buildStatus/icon?job=test_proxy_develop)](http://ci.theforeman.org/job/test_proxy_develop/)
+[![Code Climate](https://codeclimate.com/github/theforeman/smart-proxy/badges/gpa.svg)](https://codeclimate.com/github/theforeman/smart-proxy)
+[![Issue Stats](http://issuestats.com/github/theforeman/smart-proxy/badge/pr)](http://issuestats.com/github/theforeman/smart-proxy)
+[![Support IRC channel](https://kiwiirc.com/buttons/irc.freenode.net/theforeman.png)](https://kiwiirc.com/client/irc.freenode.net/?#theforeman)
+
+[Smart Proxy](http://projects.theforeman.org/projects/smart-proxy/wiki) is a free open source project that provides restful API to subsystems such as DNS, DHCP, etc, for a higher level orchestration tools such as [Foreman](https://github.com/theforeman/foreman).
```
Comment (dLobatog) [inline README.md]: `for higher level orchestration tools`

File: README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+# Configuration
+Read the [Smart Proxy Settings section](http://theforeman.org/manuals/1.8/index.html#4.3.2SmartProxySettings) of the manual.
+
+# For Developers
+* [API Reference](http://projects.theforeman.org/projects/smart-proxy/wiki/API)
+* Smart Proxy Plugin development [how-to] (http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Smart-Proxy_Plugin)
+
+# Special thanks
+
+The original author of this project is [Ohad Levy](http://github.com/ohadlevy)
```
Comment (dLobatog) [inline README.md]: If you meant to put this in separate lines I think you have to put an empty line in between these.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+ * Puppet - Any Puppet server from 0.24.x
+ * Puppet CA - Manage certificate signing, cleaning and autosign on a Puppet CA server
+ * Realm - Manage host registration to a realm (e.g. FreeIPA)
+ * TFTP - any UNIX based tftp server
+
+# Installation
+Read the [Smart Proxy Installation section](http://theforeman.org/manuals/1.8/index.html#4.3.1SmartProxyInstallation) of the manual.
+
+# Configuration
+Read the [Smart Proxy Settings section](http://theforeman.org/manuals/1.8/index.html#4.3.2SmartProxySettings) of the manual.
```
Comment (domcleal) [inline README.md]: use `latest` rather than versions I'd say

**Comment by dmitri-d** [comment]
@logicminds: updated  development docs at http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Smart-Proxy_Plugin. @domcleal, @eLobato: fixed issues you pointed out.

**Comment by dLobatog** [comment]
Merged as c08eb9a80d33655279ad4cc6039f0299ccb14e3b, thanks @witlessbird !

**Pull request closed without merging**